misc/config_tools/configurator/README.md

```diff
@@ -78,15 +78,17 @@ make configurator
 Run following command in the 'acrn-hypervisor' directory.
 
 ```shell
-cd misc/config_tools
+cd misc\config_tools
 python scenario_config/schema_slicer.py
 python scenario_config/jsonschema/convert.py
 xmllint --xinclude schema/datachecks.xsd > schema/allchecks.xsd
 
 python -m build
+del .\configurator\packages\configurator\thirdLib\acrn_config_tools-3.0-py3-none-any.whl
```

Comment:
Should we use Linux-style commands in this guide?

Reply:
This in in the windows section of the instructions. I tried to keep to the same slash style that was there.

Reply:
Windows accepts both styles, and there are more forward slashes on row 82, 83, 84 and 90. How about we keep them as they are?

Reply:
done